src/coreclr/gc/gcinterface.h

```diff
@@ -652,6 +652,10 @@ class IGCHeap {
     // Get the last memory load in percentage observed by the last GC.
     virtual uint32_t GetMemoryLoad() = 0;
 
+    // Get the total paused duration 
+    // AndrewTodo: Does the order of methods in this interface matters?
+    virtual uint64_t GetTotalPauseDuration() = 0;
```

Comment:
let's make sure to not add methods in the middle so clrgc.dll can still work with previous versions of coreclr.dll.